### PR TITLE
Mutli_asic check for T2_arppopulate qos_test_corrected

### PR DESCRIPTION
**Pull request by ansrajpu-git**
### Description of PR

The check - if the chassis is multi_asic , should be verified with the num_asic count on the dut.
Adding the correct method to verify multi_asic chassis in sai_qos_tests 
In continuation to PR #11282 
Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205
- [ ] 202305
- [ ] 202311

### Approach
For verifying the chassis is multi_asic ,the num_asic count on the dut should be greater than 1.

#### What is the motivation for this PR?
Found issue -
https://github.com/sonic-net/sonic-mgmt/issues/11236

#### How did you verify/test it?
Executed qos  tests on t2 chassis & verified.

### CODE REVIEW

**Comment by judyjoseph** [comment]
Hi @arista-nwolfe could you review this PR as well - to make sure that all cases are covered even multi-dut

**Comment by judyjoseph** [comment]
including  @arlakshm

File: tests/qos/qos_sai_base.py [hunk 1]
```diff
@@ -807,6 +807,10 @@ def dutConfig(
         dst_dut = get_src_dst_asic_and_duts['dst_dut']
         src_mgFacts = src_dut.get_extended_minigraph_facts(tbinfo)
         topo = tbinfo["topo"]["name"]
+        multi_asic_dut = False
+        if get_src_dst_asic_and_duts['src_dut'].sonichost.is_multi_asic \
+                and get_src_dst_asic_and_duts['dst_dut'].sonichost.is_multi_asic:
+            multi_asic_dut = True
```
Comment (arista-nwolfe) [inline tests/qos/qos_sai_base.py:813]: This still doesn't cover the case where you have `multi_dut` variant and only one of the `src_dut` or `dst_dut` is multi_asic. In that case you need to run the `sudo ip netns exec asic{}` versions of the commands on the multi-asic dut but not on the single-asic dut.

Comment (ansrajpu-git) [reply tests/qos/qos_sai_base.py]: @arista-nwolfe ,changes are made to cover both the single_asic  & multi_asic T2 broadcom-dnx devices for arp_populate test. Please review. 
Ping is needed make sure the Arp doesn't age out before qos test run completes.

**Review by arista-nwolfe — COMMENTED** [review]
Latest change looks good to me

File: tests/qos/qos_sai_base.py (hunk 1 above)
Comment (arista-nwolfe) [reply tests/qos/qos_sai_base.py]: Thanks, new change looks good to me

**Review by arista-nwolfe — APPROVED** [review]

**Comment by arlakshm** [comment]
Hi @ansrajpu-git, can you fix the precommit failures

**Comment by arista-nwolfe** [comment]
I tried this change out on one of our multi-asic/single-asic mixed systems and I didn't see any failures

**Review by arlakshm — APPROVED** [review]

**Pull request merged**